CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Improve `pod install` performance for pods with exact file paths rather than glob patterns  
+  [Muhammed Yavuz Nuzumlalı](https://github.com/manuyavuz)
+  [#7473](https://github.com/CocoaPods/CocoaPods/issues/7473)
```

Comment:
this should be `pull/7473`

Reply:
Sorry, I just copied from danger template 🤦🏻‍♂️